### ISSUE TITLE
https://sequelize.org unreachable - DNS Issues

### ISSUE THREAD

**Issue by phuze** [sequelize/website#121]
https://sequelize.org is unreachable, citing a DNS misconfiguration.

![image](https://user-images.githubusercontent.com/9218035/164370184-98c8fa7d-00fc-42f3-ac9b-9d6ea4cb49b3.png)

Here's what's currently being reported for the domain:

![image](https://user-images.githubusercontent.com/9218035/164370629-d9cc4f29-8f01-43da-a71f-54232a93f144.png)

It looks like the sequelize.org [website](https://github.com/sequelize/website) is hosted on GitHub pages, which means your DNS A records should be pointing to:

```
185.199.108.153
182.199.109.153
182.199.110.153
182.199.111.153
```

You can read more about how to configure the website on the [GitHub documentation](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site).

**Comment by Neel278** [comment]
Great way to actually describe an issue. Really great learning for me. I just posted my first issue so made some mistakes and also provided very less info but will make sure to do better next time. Again thanks!

**Comment by WikiRik** [comment]
Yes, we had some issues with importing our Cloudflare config but that's all resolved now.

I will however keep this issue open since https://sequelizejs.com/ has not been fixed yet

**Comment by WikiRik** [comment]
https://sequelizejs.com/ should now also be fixed, so I'm closing this issue
